ci/test/00_setup_env_arm.sh

```diff
@@ -18,3 +18,4 @@ export GOAL="install"
 # -Wno-psabi is to disable ABI warnings: "note: parameter passing for argument of type ... changed in GCC 7.1"
 # This could be removed once the ABI change warning does not show up by default
 export BITCOIN_CONFIG="-DREDUCE_EXPORTS=ON -DCMAKE_CXX_FLAGS='-Wno-psabi -Wno-error=maybe-uninitialized'"
+export BITCOIND=bitcoin-node # Used in functional tests
```

Comment:
In commit "build: depends makes libmultiprocess by default" (48268facc9bb4f3e4f98cff5c21e1fe490b232e2)

This isn't actually doing anything because RUN_FUNCTIONAL_TESTS is false above

Reply:
Moved to "CentOS, dash, gui".

Reply:
(it snuck back in, but it's now also dropped from `00_setup_env_arm.sh`)